src/Project/Project.jl

```diff
@@ -0,0 +1,427 @@
+    stored in the projectDictionary, and plotting data, and responds to enableNotifications
+    of changes.
+=#
+mutable struct Project
+    name::String
+    projectDirectory:: String
+    projectDictionary::Dict{String,Any}
+    plt::Any #For the plot
+    plotOptions::Int # = combinations of MODEL, GRID, MESH
+#
+#   For drawing
+#
+    outerBndryPoints       ::Array{Any}              # CHAIN
+    outerBndryNames        ::Array{String}
+    innerBoundaryPoints    ::Array{Any}              # Array of CHAINs
+    innerBoundaryNames     ::Array{Any}              # Array of CHAINs of names
+    innerBoundaryChainNames::Array{String}           # Array of the names of the CHAINs
+    refinementRegionPoints ::Array{Array{Float64,2}} # Array of Array of points
+    refinementRegionNames  ::Array{String}
+    refinementRegionLoc    ::Array{Array{Float64}}   # Center point of a refinement region
+    bounds                 ::Array{Float64}
+    userBounds             ::Array{Float64}
+    xGrid                  ::Array{Float64}
+    yGrid                  ::Array{Float64}
+    xMesh                  ::Array{Float64}
+    yMesh                  ::Array{Float64}
+    backgroundGridShouldUpdate::Bool
+    meshShouldUpdate        ::Bool
+end
+
+const defaultPlotPts  = 50
+const meshFileFormats = Set(["ISM", "ISM-V2", "ABAQUS"])
+const plotFileFormats = Set(["sem", "skeleton"])
+const smootherTypes   = Set(["LinearSpring", "LinearAndCrossbarSpring"])
+const statusValues    = Set(["ON", "OFF"])
+const refinementTypes = Set(["smooth", "sharp"])
+
+
+"""
+    openProject(fileName::String, folder::String)
+
+Open existing project described in the control File.
+
+    folder   = folder the control file is in
+    fileName = the name of the file
+"""
+function openProject(fileName::String, folder::String)
+
+    controlFile = joinpath(folder,fileName)
+    controlDict = ImportControlFile(controlFile)
+
+    # Strip off the file path into a temporary name
+    tempName = splitdir(fileName)
+    # Separate the project name from `.control`
+    splitName = split(tempName[2], ".")
+    # Pull the correct project name
+    s = string(splitName[1]) # This is dumb
+
+    # Instantiate an empty project with some defaults
+    proj = newProject(s,folder)
+#
+#   Overwrite defaults
+#
+    proj.projectDictionary = controlDict
+
+    assemblePlotArrays(proj)
+    clearUndoRedo()
+
+    return proj
+end
+
+
+"""
+    saveProject(proj::Project)
+
+    proj     = Project to be saved
+Save a project dictionary to the file path specified when the project was created.
+"""
+function saveProject(proj::Project)
+    fileName = joinpath(proj.projectDirectory,proj.name)*".control"
+    WriteControlFile(proj.projectDictionary,fileName)
+end
+
+
+"""
+    newProject(name::String, folder::String)
+
+Create a new project with the given name. That name will be used
+for the mesh and plot files in the specified folder.
+"""
+function newProject(name::String, folder::String)
```

Comment:
As above: Should be just a constructor call:
```suggestion
function Project(name::String, folder::String)
```
However, I see that dispatch is of no use here. One possibility: Check if the given name + folder combination already exists: If yes, open it, if not, create a new one. One could additionally use a kwarg such as `Project(...; overwrite=false)` which you can set to `true` to create a new project even if there's an existing one with the same name.

Reply:
Nope. Verb+noun.